stylus/ui-components/modals.styl

```diff
@@ -34,7 +34,6 @@ $modals
         display           block
         box-sizing        border-box
         border-radius     em(8px)
```

Comment:
I don't get why you remove this line. Without it the modal would take full width of the viewport